### ISSUE TITLE
keras报错TypeError:softmax() got an unexpected keyword argument 'axis'的解决办法 - sqzhang's Blog

### ISSUE THREAD

**Issue by shengqiangzhang** [shengqiangzhang/shengqiangzhang.github.io#63]
https://shengqiangzhang.github.io/2019/03/26/keras%E6%8A%A5%E9%94%99%E7%9A%84%E8%A7%A3%E5%86%B3%E5%8A%9E%E6%B3%95/ 

 Every failure is leading towards success.
